.changeset/evil-streets-boil.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-admin": minor
+---
+
+Add `NotFound` Component and implement NotFound Fallback in MasterMenuRoutes
```

Comment:
Should we add the NotFound component to `@comet/admin` instead? Then it could be added to `Stack` as well.